code/game/machinery/anomaly_pad.dm

```diff
@@ -0,0 +1,125 @@
+///Magical pad that suspends anomalies. Simply apply an anomaly core and it will make the anomaly fly above it
+/obj/machinery/anomaly_pad
+	name = "anomaly pad"
+	desc = "If an anomaly core is applied, this machine will keep it stable and suspended. Looks beautiful and cannot possibly go wrong."
+
+	icon = 'icons/obj/machines/anomaly_pad.dmi'
+	icon_state = "pad"
+
+	circuit = /obj/item/circuitboard/machine/anomaly_pad
+	density = TRUE
+
+	use_power = IDLE_POWER_USE
+	power_channel = EQUIP
+
+	///this is the big beautiful anomaly floating ontop of us
+	var/obj/effect/anomaly/anomaly
+	///our base active power use. Decreases with laser efficiency and is * 0.1 for inactive
+	var/base_power = 500
+
+/obj/machinery/anomaly_pad/RefreshParts()
+	..()
```

Comment:
```suggestion
	. = ..()
```

Reply:
It doesn't expect a return value

Reply:
We tend to save the return value nonetheless as a convention. Makes it easier for mass-replacements or for changing the logic down the line, without having to worry if a stray proc forgot to do it.

Reply:
Alright then, I didn't do I thought it was misleading. This one didn't actually have a sensible parent call, but I'll change the others then